src/components/progress-tracker/progress-tracker.stories.tsx

```diff
@@ -102,6 +102,44 @@ export const DefaultLabelValueLabelsPositionBottom: ComponentStory<
   );
 };
 
+export const DefaultLabelValueLabelsPositionLeft: ComponentStory<
```

Comment:
I've left the story as it is and added a test story with a few different `labelWidth` options. In doing this I noticed the width of the label wasn't behaving as intended, hence adding `flex-grow` & `flex-shrink` to set the width of the bar and the label so that the bar just takes up the remaining space.